### ISSUE TITLE
cdiomodule.h needs patching for newer libcdio-paranoia

### ISSUE THREAD

**Issue by ryuo** [tuffy/python-audio-tools#48]
This is busted on newer versions of it. The headers have moved into a different directory. This is apparent on ArchLinux and DragonFly BSD. Here is the patch I hacked up to get the software to compile:

--- src/cdiomodule.h.orig       2015-06-06 21:03:42 UTC
+++ src/cdiomodule.h
@@ -1,6 +1,6 @@
 #include <Python.h>
-#include <cdio/cdda.h>
-#include <cdio/paranoia.h>
+#include <cdio/paranoia/cdda.h>
+#include <cdio/paranoia/paranoia.h>
 #include "array.h"
 
 /********************************************************

**Comment by tuffy** [comment]
It looks like this change occurred starting with cdio-paranoia version 0.90.  I've added a change which uses pkg-config to detect the library's version and use the proper pair of imports depending on which version if cdio-paranoia is found.  Hopefully that will fix the build to work under both newer and older versions of the library.

**Comment by sl1pkn07** [comment]
Hi

in arch, the version of libcdio-paranoia is a mixed of numbers and symbols (like upstream)
http://ftp.gnu.org/gnu/libcdio/

Versión                 : 10.2+0.93+1-1

the setup.py always detect this as OLD_PARANOIA

greetings

**Comment by tuffy** [comment]
I've pushed out a fix which should check for different libcdio paranoia libraries more reliably and it builds correctly for me under more recent versions such as those used by Arch and Manjaro.

**Comment by sl1pkn07** [comment]
yes. fixed in https://github.com/tuffy/python-audio-tools/commit/c86deabd16888b80a93f39828552e281df7fa8c0 XD

greetings
